src/main/java/codeu/model/data/User.java

```diff
@@ -88,4 +94,30 @@ public String getAbout() {
   public void setAbout(String aboutMessage) {
     this.about = aboutMessage;
   }
+
+  /** Returns the conversations in which the user has sent a message. */
+  public LinkedHashMap<UUID, Boolean> getConversations() {
+    return conversations;
+  }
+
+  /** Adds a conversation to the list */
+  public void addConversation(UUID conversationId) {
+    if (!conversations.containsKey(conversationId)) {
+      conversations.put(conversationId, true);
```

Comment:
If you negate this, you will never add the conversation back. 

Moreover, you can use Java 8 for this:

`conversations.computeIfPresent(conversationId, (k, v) -> true);`

Reply:
I don't mean to bring hidden conversations back with this function but with `resetConversations()` instead, which is working fine. I also think `computeIfPresent()` wouldn't be able to add new conversations to the map, it can only change those already present back to true. I think I can use `computeIfAbsent()` for what I have right now, unless you want me to make this function also bring back those that are hidden.